source/_introduction.html.md.erb

```diff
@@ -263,9 +270,50 @@ The access token must be sent with every API call in an HTTP authorization heade
 
 The header name is **Authorization** and the value of the header is **Bearer ll352u9jujauoqz4gstvsae05**. Since the access token is being transmitted in clear text, all API calls are done over HTTPS.
 
-<aside class="warning">The token generated runs with your credentials. Be sure to secure your tokens and do not share tokens with anyone.</aside>
+<aside class="notice">A best practice is to use a shared account, such as admin@company.com, rather than your individual work account.</aside>
+
+### Adding Security by Using OAuth
+
+If your application requires users to be able to login with their own account, you must implement the full OAuth flow. 
+
+At a high level, this involves the following steps:
+
+1. [Register as a Developer](https://developers.smartsheet.com/register) with Smartsheet so you can get access to the Developer Tools menu.
+2. Register your app to get both a clientId and client secret, which you'll need to store securely. Some best practices suggestions for security are below.
+3. Go through the steps in the [Third-party App Development](#third-party-app-development) section once you are ready to request authorization.
+
+### Security Best Practices
+
+The following provides some best practices to consider when working with the Smartsheet API and any API keys, tokens, or other sensitive information.
+
+#### Source Code and Version control
+
+Be wary of committing API keys to accessible version control systems like GitHub or BitBucket.
 
-<aside class="notice">If your application requires users to be able to login with their own account, you must implement the full OAuth flow. See the documentation for <a href="#third-party-app-development">Third-party App Development</a> and <a href="https://developers.smartsheet.com/register">Developer Registration</a>.</aside>
+* Use server environment variables. 
```

Comment:
Would drop server from this to just, `use environment variables`. Would put a note in here that if using one of the SDKs, the environment variable SMARTSHEET_ACCESS_TOKEN can be used in place of passing in an API token to the Smartsheet client initialization/client builder code. If the access token is null on input to the client builder, the SDK will automatically pick up the value of the SMARTSHEET_ACCESS_TOKEN environment variable.